test/katex-spec.js

```diff
@@ -88,115 +87,100 @@ const parseAndSetResult = function(expr, result, settings) {
+                pass: true,
+                message: "'" + actual + "' succeeded in building",
             };
+
+            expect(actual).toParse(usedSettings);
+
+            try {
+                _getBuilt(actual, settings);
+            } catch (e) {
+                result.pass = false;
+                if (e instanceof ParseError) {
+                    result.message = "'" + actual + "' failed to " +
+                        "build with error: " + e.message;
+                } else {
+                    result.message = "'" + actual + "' failed " +
+                        "building with unknown error: " + e.message;
+                }
+            }
+
+            return result;
         },
 
-        toParseLike: function(util, baton) {
-            return {
-                compare: function(actual, expected, settings) {
-                    const usedSettings = settings ? settings : defaultSettings;
-
-                    const result = {
-                        pass: true,
-                        message: "Parse trees of '" + actual +
-                            "' and '" + expected + "' are equivalent",
-                    };
-
-                    const actualTree = parseAndSetResult(actual, result,
-                        usedSettings);
-                    if (!actualTree) {
-                        return result;
-                    }
-                    const expectedTree = parseAndSetResult(expected, result,
-                        usedSettings);
-                    if (!expectedTree) {
-                        return result;
-                    }
-                    stripPositions(actualTree);
-                    stripPositions(expectedTree);
-                    if (!util.equals(actualTree, expectedTree, baton)) {
-                        result.pass = false;
-                        result.message = "Parse trees of '" + actual +
-                            "' and '" + expected + "' are not equivalent";
-                    }
-                    return result;
-                },
+        toParseLike: function(actual, expected, settings) {
+            const usedSettings = settings ? settings : defaultSettings;
+
+            const result = {
+                pass: true,
+                message: "Parse trees of '" + actual +
+                    "' and '" + expected + "' are equivalent",
             };
-        },
 
+            const actualTree = parseAndSetResult(actual, result,
+                usedSettings);
+            if (!actualTree) {
+                return result;
+            }
+            const expectedTree = parseAndSetResult(expected, result,
+                usedSettings);
+            if (!expectedTree) {
+                return result;
+            }
+
+            stripPositions(actualTree);
+            stripPositions(expectedTree);
+
+            if (JSON.stringify(actualTree) !== JSON.stringify(expectedTree)) {
+            // if (!util.equals(actualTree, expectedTree, baton)) {
```

Comment:
Remove commented out code.  While `jest` has util functions, it doesn't have a util for checking equality.